catalyst_voices_packages/catalyst_cardano_serialization/lib/src/types.dart

```diff
@@ -70,3 +73,240 @@ extension type const SlotBigNum(int value) {
   /// Serializes the type as cbor.
   CborValue toCbor() => CborSmallInt(value);
 }
+
+/// Represents the balance of the wallet in terms of [Coin].
+///
+/// For now, other assets than Ada are not supported and are ignored.
+final class Value extends Equatable {
```

Comment:
Can we think of a better name than `Value`? Value is too generic and can be misleading.
Based on the comments to the class probably we can use `Balance`.

Reply:
The name comes from official CIP-30 docs but I agree, it's too vague, renamed to Balance.

Reply:
We don't need to follow bad code practices ;)